rules/use-vertical-text/rule.md

```diff
@@ -35,12 +35,18 @@ To
 ![Figure: Bad example - Not using vertical text for headings, when you have lots of thin columns](RS_VerticalText_1.gif)  
 :::
 
-::: bad  
-![Figure: Bad example - Not using ticks and crosses](RS_VerticalText_2.gif)  
+::: good  
```

Comment:
Hey @jeoffreyfischer 

We have an OK class to be used here ;)

Reply:
Done - changed "good" to "ok"